lib/mqtt.ts

```diff
@@ -66,18 +71,123 @@ export interface MqttSubscribeRequest extends MqttRequest {
 
 type Payload = string | Object | DataView;
 
-export class Connection {
+export class AwsIotMqttConnectionConfigBuilder {
```

Comment:
I feel like this doesn't belong here, it belongs in the IoT SDK.

Reply:
eeeh, let's revisit this later. I did move it to a different file though.